spec/factories/activities.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :activity do
+    actor
+    object { create(:user) }
+    event 'test'
+    notifier_type 'test_notifier'
```

Comment:
If the schema says it's not null then there is a validation that checks that it is not empty.